packages/embroider-css-modules/package.json

```diff
@@ -61,6 +61,7 @@
     "@babel/plugin-proposal-class-properties": "^7.18.6",
     "@babel/plugin-proposal-decorators": "^7.21.0",
     "@babel/preset-typescript": "^7.21.4",
+    "@babel/runtime": "^7.21.5",
```

Comment:
I added `@babel/runtime` to `embroider-css-modules` (a TS project) just in case.